includes/class-sensei-wp-kses.php

```diff
@@ -112,11 +112,11 @@ public static function get_source_html_tag_allowed_attributes() {
 	 * @param array  $allowed_html
 	 * @return string Content
 	 */
-	public static function maybe_sanitize( $content, $allowed_html ) {
-		$html_security = ! Sensei()->settings->get( 'sensei_video_embed_html_sanitization_disable' );
+public static function maybe_sanitize( $content, $allowed_html ) {
```

Comment:
Hmm, I think this linting issue originates from the method before this one ([line 93](https://github.com/Automattic/sensei/blob/54199752bbf658c51973f9ea06b403536115e012/includes/class-sensei-wp-kses.php#L93)). Could you try adding a break after the closing comment on line 93 and keep the tab indent on the methods that you've changed?